digital_asset_types/src/dapi/common/asset.rs

```diff
@@ -151,38 +151,23 @@ pub fn safe_select<'a>(
         .and_then(|v| v.pop())
 }
 
-#[cfg(feature = "fetch-raw-fields")]
-async fn process_raw_fields(
-    name: Option<String>,
-    symbol: Option<String>,
-    asset_data: &AssetData,
+fn process_raw_fields(
+    name: &Option<Vec<u8>>,
+    symbol: &Option<Vec<u8>>,
 ) -> (Option<String>, Option<String>) {
-    let name_result = if let Some(name) = &name {
-        let raw_name = asset_data.raw_name.clone();
-        match String::from_utf8(raw_name) {
-            Ok(s) => Some(s),
-            Err(_) => Some(name.clone()),
-        }
-    } else {
-        None
-    };
-
-    let symbol_result = if let Some(symbol) = &symbol {
-        let raw_symbol = asset_data.raw_symbol.clone();
-        match String::from_utf8(raw_symbol) {
-            Ok(s) => Some(s),
-            Err(_) => Some(symbol.clone()),
-        }
-    } else {
-        None
-    };
-
+    let name_result = name
+        .as_ref()
+        .and_then(|name| String::from_utf8(name.clone()).ok());
+    let symbol_result = symbol
+        .as_ref()
+        .and_then(|symbol| String::from_utf8(symbol.clone()).ok());
     (name_result, symbol_result)
 }
 
 pub fn v1_content_from_json(
     asset_data: &asset_data::Model,
     cdn_prefix: Option<String>,
+    raw_data: Option<bool>,
```

Comment:
Why is this being added here? I don't follow the logic. The raw data is for the on-chain data, not the off-chain data. This method shouldn't be touched.